iree/compiler/Conversion/LinalgToLLVM/LinalgVectorizePass.cpp

```diff
@@ -82,6 +82,12 @@ void LinalgVectorizationPass::runOnFunction() {
       op->replaceAllUsesWith(contract);
   });
 
+  // Lowering transfer ops before unrolling as unrolling doesn't support
+  // transpose.
+  OwningRewritePatternList transferOpLowering(&getContext());
+  vector::populateVectorTransferLoweringPatterns(transferOpLowering);
+  (void)applyPatternsAndFoldGreedily(funcOp, std::move(transferOpLowering));
```

Comment:
I think @GMNGeoffrey wanted us to follow-up on these casts to `void`, but they shouldn't stop this merge.

Reply:
Sure, @GMNGeoffrey let me know what you think. I can fix it although it is odd to just fix this spot. Also I'm hoping to remove this piece of code in the near future as it was a workaround to unblock integration.